feat: docs on AUTHORS/CONTRIBUTORS files (#326)
ansys/pyansys-dev-guide: doc/source/packaging/structure.rst

The ``AUTHORS`` file
--------------------
An ``AUTHORS`` file is used to specify the authorship of the repository. Its
format has been defined by the Ansys Legal department. External contributors
may be added on demand to this file. Make sure to adapt the project name on

> your specific repository ``AUTHORS`` file.

Reviewer: ```suggestion
your specific repository's ``AUTHORS`` file.
```

   :language: markdown

The ``CONTRIBUTORS.md`` file
----------------------------

> The ``CONTRIBUTORS.md`` file is used to list the contributors to the repository. It is
> used to credit the authors for their individual contributions and provide a
> record of authorship for the codebase. Use your first and last name and provide
> a link to your GitHub username.

Reviewer: ```suggestion
The ``CONTRIBUTORS.md`` file is used to list the contributors to the repository. Its
purpose is to credit the authors for their individual contributions and provide a
record of authorship for the codebase. Provide your first and last names and
a link to your GitHub username.
```
